core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1560,6 +1562,15 @@ public Text getMaxRow(String tableName, Authorizations auths, Text startRow,
     return FindMax.findMax(scanner, startRow, startInclusive, endRow, endInclusive);
   }
 
+  @Override
+  public Text getMaxRow(String tableName, Authorizations auths, RowRange rowRange)
```

Comment:
The other getMaxRow method should call this getMaxRow method, creating a RowRange from the args.

Reply:
Addressed in 3a6d615